src/core/selection/selection.view.js

```diff
@@ -194,11 +195,21 @@ export default class SelectionView extends View {
 	}
 
 	select(items) {
-		if (this.selection.mode === 'range') {
-			this.selectionState.clear();
-			this.selectionState.toggle(items, true);
-		} else {
-			this.selectionState.toggle(items);
+		if (arguments.length && !isUndefined(items)) {
```

Comment:
the problem is that isUndefined shouldn't be here but as we use lambdas for now in commands i just can't pass ...argumnets, so this is notification to refactor in the future